app/src/backend/billing/index.ts

```diff
@@ -157,3 +157,21 @@ export const fetchBillingInformation = async (billingId: string) => {
     return res.data;
   });
 };
+
+export const revokeBillingTeamInvite = async (inviteId: string, email: string) => {
+  if (!inviteId || !email) {
+    return null;
+  }
+  const revokeInvite = httpsCallable(getFunctions(), "billing-revokeBillingTeamInvite");
+  return revokeInvite({ inviteId, userEmail: email });
+};
+
+export const inviteUsersToBillingTeam = async (billingId: string, userEmails: string[]) => {
+  if (!billingId || !userEmails) {
+    return null;
+  }
+
+  const inviteUsers = httpsCallable(getFunctions(), "billing-createBillingTeamInvites");
+
+  return inviteUsers({ userEmails, billingId });
+};
```

Comment:
re-evaluate if we can directly use the existing createInvite and revokeInvite backend functions.
Mentioned reasons in the backend PR

Reply:
not re-thinking existing apis